client/lib/signup/test/dependency-store.js

```diff
@@ -23,7 +23,7 @@ describe( 'dependency-store', function() {
 		SignupDependencyStore = require( '../dependency-store' );
 		SignupActions = require( '../actions' );
 
-		const store = createStore( reducer );
+		const store = createStore( combineReducers( initialReducers ) );
```

Comment:
These are test functions, and probably shouldn't even be bringing in all initalReducers anyway. They should probably only add the reducer they care about to their test store. I was just trying not to rewrite everything while implementing some changes. 😛  I figure this PR is big enough already.